recipe/meta.yaml

```diff
@@ -42,9 +42,9 @@ requirements:
     - setuptools
     - cython >=0.24.0
     - fastrlock >=0.3
-    - cudnn
+    - cudnn <8.1  # [cuda_compiler_version != "11.1"]
```

Comment:
OK so I am fixing a mistake I made earlier today. For CuPy v8.4.0/v9.0.0b2 there was a max version cap for cuDNN, and v8.1 was (mistakenly) considered too high, so in all builds the cuDNN support was not built. 

This change restores the cuDNN support for CUDA < 11.1. We still can't build it for CUDA 11.1, because cuDNN v8.1 is the only available version on CF that supports CUDA 11.1+. The version cap is removed in CuPy v8.5.0/v9.0.0b3 (cupy/cupy#4683), so we can restore it when building the new versions.